src/allmydata/scripts/stats_gatherer.py

```diff
@@ -7,6 +7,10 @@
 class CreateStatsGathererOptions(BasedirOptions):
     default_nodedir = None
 
+    optParameters = [
+        ("pickledir", "p",  None, "Path where the statistics will be recorded."),
```

Comment:
-p for --pickledir conflicts with -p for --port. I suggest removing short -p for this command.